internal/campaign/campaign.go

```diff
@@ -60,10 +68,13 @@ func OpenCampaign() (*Campaign, error) {
 	}, nil
 }
 
-func readReposTxtFile() ([]Repo, error) {
-	file, err := os.Open("repos.txt")
+func readReposTxtFile(filename string) ([]Repo, error) {
```

Comment:
Should we perhaps log the name of the file that's going to be loaded? 

I get the sense that one failure mode for this feature could be users accidentally using (or not using) the intended repos file.

Logging the filename (maybe in a colour that stands out) would be one way to give people a chance to notice their mistake.

WDYT?

Reply:
💯  agreed.
Good shout.

Reply:
On second thought; I think we should work towards a larger refactor of the logging.
Currently the loggers are constructed from the commands (to interact with its Stdout). It's fine, but it's a problem when we want to log things from internal packages. Passing the logger as a parameter is not elegant.

I'll open a ticket to refactor it, perhaps following some singleton pattern. We'd still init it from the command itself but could have a `logging.GetLogger()` that we can retrieve from the internal packages.

Reply:
WDYT @rnorth

Reply:
Seems like a good idea - definitely as a separate ticket!

Reply:
Resolving thread, filed #79 to take care of the logging thing a bit later.

Reply:
👍 that's a good idea